data/complex/exponential.lean

```diff
@@ -673,6 +685,10 @@ by simp [real.exp]
 lemma exp_add : exp (x + y) = exp x * exp y :=
 by simp [exp_add, exp]
 
+lemma exp_mul (x : ℝ) : ∀ n : ℕ, exp(n*x) = (exp(x))^n
+| 0 := by rw [nat.cast_zero, zero_mul, exp_zero, pow_zero]
+| (nat.succ n) := by rw [pow_succ', nat.cast_add_one, add_mul, exp_add, ←exp_mul, one_mul]
+

```

Comment:
This should probably be called `exp_nat_mul`

Reply:
Maybe with the real power stuff. Do you know why the build isn't compiling? It works on my computer.

Reply:
You should merge the latest version of mathlib